examples/platforms/posix/openthread-core-posix-config.h

```diff
@@ -136,4 +136,25 @@
 #ifndef OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_METRICS_ENABLE
 #define OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_METRICS_ENABLE 1
 #endif
+
+/**
+ * @def RADIO_CONFIG_SRC_MATCH_SHORT_ENTRY_NUM
+ *
+ * The number of short source address table entries.
+ *
+ */
+#ifndef RADIO_CONFIG_SRC_MATCH_SHORT_ENTRY_NUM
+#define RADIO_CONFIG_SRC_MATCH_SHORT_ENTRY_NUM 10
```

Comment:
Possible to define as `OPENTHREAD_CONFIG_MLE_MAX_CHILDREN`?

Reply:
Updated.